styleguide.config.js

```diff
@@ -320,7 +320,7 @@ module.exports = {
   pagePerSection: true,
   showCode: true,
   showUsage: true,
-  serverPort: Number(process.env.PORT),
+  serverPort: 4040,
```

Comment:
@aldeed I noticed you changed the serverPort from a Number to a dynamic `process` one. I changed this back to `4040` because apparently: 1) the `styleguide:build` step requires running through this config file and 2) when running the build step in the CircleCI container, it doesn't have access to `process.env`. 

Here is the error: https://circleci.com/gh/reactioncommerce/reaction-component-library/1409

```
$ styleguidist build
Something is wrong with your style guide config

serverPort config option should be number, received number.

Example:

{
  serverPort: 6060
}

Learn how to configure your style guide:
https://react-styleguidist.js.org/docs/configuration.html

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
The command '/bin/sh -c yarn run styleguide:build' returned a non-zero code: 1
Exited with code 1
```

I wasn't sure of any other ways to get around this.